web/src/styles/base/margin-padding.scss

```diff
@@ -27,3 +27,8 @@
 .mla {
   margin-left: auto !important;
 }
+
+.bottom-radius-lg {
```

Comment:
dust - it's kind of odd that the border at the bottom has a radius, but the border at the top does not. From the other designs that I've seen, there is a radius applied across the entire element. Separately, we do try to use USWDS for radiuses can we use one of those instead? i.e. radius-bottom-lg

https://designsystem.digital.gov/utilities/border/#border-radius

Reply:
I can pull from USWDS, thanks for the callout! As far as only the bottom having padding, there are two separate parts, and the border between them does not have a border-radius.